okrs/2019-q3.md

```diff
@@ -0,0 +1,27 @@
+## Objective: Projects that depend on IPLD are using new interfaces and specifications
+
+* Complete unixfsv2 implementation and relevant `iq` functionality (@mikeal)
+* Document `iq` and write a blog post (@mikeal)
+* Complete selector work necessary for GraphSync (@warpfork)
+* TODO: go-ipld-prime OKR's
+* TODO: schema OKR's
+
+## Objective: Building applications on IPLD is simple and fun
+
+* Build a delicio.us clone w/ IPLD (@mikeal)
+* Complete STAC geo data IPLD demo (@vmx)
+* TODO: what sample applications would @rvagg and @warpfork like to build?
```

Comment:
Two things on my list of possible experiments

* Wikipedia article search
* Lit Package manager for Luvit / Lua is a nice small data set that could do with some indexing for keywords in description, titles, authors and keywords, data is stored as JSON in git tag bodies in https://github.com/luvit/lit-backup.

Reply:
I'd find it cool to see a TiddlyWiki like thing based in IPLD, I think it would be interesting to see how e.g. create indexes for tags etc.